src/paths/components/updateAll.yaml

```diff
@@ -0,0 +1,26 @@
+summary: Update existing components
+tags:
+  - components
+operationId: componentsUpdateAll
+requestBody:
+  description: Body request that contains a list of ComponentDto objects
```

Comment:
don't use 'Body request that contains'. Describe it in the following manner: list of components to be updated. Each element of the list contains the new implementation of the component.

This remark is valid for all 'update', 'updateAll' and 'create' paths having a requestBody